packages/styles/src/navigation.scss

```diff
@@ -1023,4 +1030,56 @@ $navListContainerSubmenu: #{$block}__list-container--submenu;
       --fdNavigation_Link_Back_Indicator_Icon: "\e1ed";
     }
   }
+
+  @include _fd-vertical-nav() {
+    .#{$block} {
+      &__item {
+        &--with-expander {
+          position: relative;
+
+          .#{$navLink} {
+            --fdNavigation_Link_Padding_Right: 2.1975rem;
+          }
+
+          .#{$navExpander} {
+            @include fd-set-position-right(var(--fdNavigation_Expander_Position_Right));
+
+            position: absolute;
+            z-index: 20;
+          }
+        }
+      }
+    }
+  }
+
+  @include _fd-horizontal-nav() {
+    .#{$block} {
+      &__item {
+        &--with-expander {
+          .fd-navigation__expander {
+            margin: 0 0.0625rem;
+            transition: var(--fdNavigation_Link_Transition);
+
+            --fdButton_Nested_Border_Radius: var(--fdNavigation_Link_Border_Radius);
+
+            @include fd-selected() {
+              background-color: var(--fdNavigation_Item_Selected_Background);
```

Comment:
since the background is changing depending on the state maybe make the background a css variable